### PR TITLE
Install packages in vscode editor container DTSW-3170

### PR DESCRIPTION
**Pull request by Tuxliri**

### CODE REVIEW

File: Dockerfile.vscode [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # syntax=docker/dockerfile:1.4
 # parameters
-ARG EXERCISE_NAME="<LX_NAME_HERE>"
-ARG DESCRIPTION="VSCode - <DESCRIPTION_HERE>"
-ARG MAINTAINER="<YOUR_FULL_NAME> (<YOUR_EMAIL_ADDRESS>)"
+ARG EXERCISE_NAME="dd21-ukf"
+ARG DESCRIPTION="VSCode - In this learning experience you will learn how to filter the raw measurements you get from the Duckiedrone's sensors."
+ARG MAINTAINER="Davide Iafrate (davide.iafrate@duckietown.com)"
```
Comment (fishrcat) [inline Dockerfile.vscode]: Revert your LX details to the template placeholders.

Comment (Tuxliri) [reply Dockerfile.vscode]: Reverted.

File: Dockerfile.vscode [hunk 2]
```diff
@@ -43,6 +48,10 @@ ENV PIP_INDEX_URL=${PIP_INDEX_URL}
 COPY --from=recipe ./dependencies-py3.* "/tmp/"
 RUN python3 -m pip install -r /tmp/dependencies-py3.txt
 
+
```
Comment (afdaniele) [inline Dockerfile.vscode]: remove extra empty line

File: Dockerfile.vscode [hunk 3]
```diff
@@ -61,4 +70,5 @@ LABEL org.duckietown.label.module.type="exercise-vscode" \
 
 # setup vscode
 COPY --from=recipe assets/vscode/setup.sh /tmp/vscode_setup.sh
-RUN /tmp/vscode_setup.sh
+RUN chmod +x /tmp/vscode_setup.sh
+RUN /tmp/vscode_setup.sh
```
Comment (afdaniele) [inline Dockerfile.vscode]: combine these two using "&&"

**Review by afdaniele — CHANGES_REQUESTED** [review]

**Comment by Tuxliri** [comment]
@afdaniele we said to move this to `v4` right?

**Review by Tuxliri — COMMENTED** [review]
Made requested changes

**Comment by afdaniele** [comment]
> @afdaniele we said to move this to `v4` right?

Yes, change it in `.dtproject`.

**Pull request merged**